.github/workflows/weekly-tasks.yml

```
@@ -5,31 +5,39 @@
 
 name: Weekly tasks
 on:
-  schedule:
-    - cron: 00 05 * * 1
+    schedule:
+        - cron: 00 05 * * 1
 
 jobs:
-  create_issue:
-    name: Create weekly issue to upgrade dependencies
-    runs-on: ubuntu-latest
-    permissions:
-      issues: write
-    steps:
-      - name:  Create weekly issue to upgrade dependencies
-        uses: imjohnbo/issue-bot@v3.3.3
-        with:
-          assignees: "espkva, henrikhermansen, joms, kennidenni, mikaila94, piofinn, saegrov, sercansercan, wkillerud"
-          rotate-assignees: true
-          labels: "dependencies"
-          project: "Kjerneteamet"
-          title: "Weekly dependency update"
-          body: |
-            ### Agenda
+    create_issue:
+        name: Create weekly issue to upgrade dependencies
+        runs-on: ubuntu-latest
+        permissions:
+            issues: write
+        steps:
+            - name: Create weekly issue to upgrade dependencies
+              uses: imjohnbo/issue-bot@v3.3.3
```

Ser nå vi har en PR som bumper denne patchen, kan ta det i samme slengen
```suggestion
              uses: imjohnbo/issue-bot@v3.3.4
```

---

- [ ] trykk på add suggestion to batch eller legg til manuelt